handlers/error.go

```diff
@@ -79,6 +79,7 @@ const (
 	ErrCreateDirCode            = "2150"
 	ErrInvalidRequestObjectCode = "2151"
 	ErrChangeK8sContextCode     = "2152"
+	ErrSavingUserPreferenceCode = "some_code"
```

Comment:
by pattern, we should have this "2153"

Reply:
Will this be automatically filled in by the area code utility, though?